users/drashna/eeconfig_users.c

```diff
@@ -6,8 +6,8 @@
 #include "eeconfig.h"
 #include <string.h>
 
-#if (TOTAL_EEPROM_BYTE_COUNT - 1) < EECONFIG_SIZE && !defined(KEYBOARD_input_club_ergodox_infinity)
-#    error "More eeprom configured than is available."
+#if !defined(KEYBOARD_input_club_ergodox_infinity2)
```

Comment:
I'm guessing that you meant this: 
```suggestion
#if !defined(KEYBOARD_input_club_ergodox_infinity)
```

Reply:
bleh, was testing....